Backend/storage/tests.py

```diff
@@ -1,3 +1,41 @@
 from django.test import TestCase
 
-# Create your tests here.
+from .models import Support, PortSpecification, Discrete, Continuous
+
+
+class SupportTestCase(TestCase):
+    def setUp(self):
+        Support.objects.create(pk="support_int", type="int", upper="inf", lower="-inf", upperInclusive="False",
+                               lowerInclusive="False")
+        Support.objects.create(pk="support_float", type="float", upper="inf", lower="-inf", upperInclusive="False",
+                               lowerInclusive="False")
+
+    def test_support(self):
+        pass
+
+
+class PortSpecificationTestCase(TestCase):
+    def setUp(self):
+        pass
+
+
+class DistributionTestCase(TestCase):
+    def setUp(self):
+        PortSpecification.objects.create(pk="mu",
+                                         name="mu - Expected number of occurrences during the given interval (mu >= 0)",
+                                         type="float", upper="inf", lower="0", upperInclusive="False",
+                                         lowerInclusive="True", optional="False")
+        Support.objects.create(pk="support_int", type="int", upper="inf", lower="-inf", upperInclusive="False",
+                               lowerInclusive="False")
+        Discrete.objects.create(name="Poisson log-likelihood", distType="discrete",
+                                url="https://www.pymc.io/projects/docs/en/stable/api/distributions/generated/pymc.Poisson.html",
+                                image_url="https://www.pymc.io/projects/docs/en/stable/_images/pymc-Poisson-1.png",
+                                output=Support.objects.get(pk='support_int'))
+
+        Discrete.objects.get(name='Poisson log-likelihood').input.set([PortSpecification.objects.get(pk='mu')])
+
+    def test_Distribution_input_output_class(self):
```

Comment:
RuntimeError: Model class Backend.storage.models.Support doesn't declare an explicit app_label and isn't in an application in INSTALLED_APPS

Reply:
@lucasgarbe @VulcanTest Haben beide den Fehler, werde den noch beheben, aber sollte keine Probleme fuers backend machen. Sind nur die Tests die nicht laufen, weil noch irgendein Import nicht laeuft. Solang das Dockerimage bei euch laeuft  koennt ihr approven. Werd den Fix fuer die Tests dann seperat, direkt auf dev branch commiten